client/components/UserDecalPanel.js

```diff
@@ -108,26 +142,47 @@ class UserDetailsPanel extends Component {
     this.setState({ checkingEligibility: false })
   }
 
+  state = {
+    formOpen: false,
+  }
+
+  _handleToggle = () => {
+    this.setState((state) => ({
+      ...state,
+      formOpen: !state.formOpen,
+    }))
+  }
+
   render () {
     const { decals } = this.props
+    console.log(this)
 
     const {
       checkingEligibility,
       redeeming,
     } = this.state
 
+    const loadingText = checkingEligibility ? 'Checking decal eligibility...' : 'Redeeming decal codes...'
+    // console.log(loadingText)
     return (
-      <ReactTable
-        className="panel user-decals"
-        columns={this.columns}
-        data={decals}
-        defaultPageSize={10}
-        loading={checkingEligibility || redeeming}
-        loadingText={checkingEligibility ? 'Checking decal eligibility...' : 'Redeeming decal codes...'}
-        manual
-        minRows={5}
-        noDataText={this._renderNoDataText()}
-        showPagination={false} />
+      // <ReactTable
+      //   className="panel user-decals"
+      //   columns={this.columns}
+      //   data={decals}
+      //   defaultPageSize={10}
+      //   loading={checkingEligibility || redeeming}
+      //   loadingText={checkingEligibility ? 'Checking decal eligibility...' : 'Redeeming decal codes...'}
+      //   manual
+      //   minRows={3}
+      //   noDataText={this._renderNoDataText()}
+      //   showPagination={false} />
+      <div className="panel user-decals">
+        <header>Decal</header>
+        <div className="panel-content">
+          {(checkingEligibility || redeeming) ? (<div className="loading">{loadingText}</div>) : (this._renderDecalCodes(decals))}
+          {(decals === []) ? (this._renderNoDataText()) : ('')}
```

Comment:
Same as above, never compare to `[]`

Reply:
I don't know what you mean by this or why, or even how to do this otherwise?

Reply:
wtf... the other one didn't get saved :/